frontend/lib/screens/new_student/new_student.dart

```diff
@@ -11,7 +12,8 @@ class NewStudent extends StatefulWidget {
 
 class _NewStudentState extends State<NewStudent> {
   bool _firstStep = true;
-  Student _newStudent = Student(name: "Bassam", surname: "Zabad");
+  //Student _newStudent = Student(name: "Bassam", surname: "Zabad");
```

Comment:
We can delete this line if we don't think it will ever been used again

Reply:
Done